packages/kit/test/apps/basics/src/routes/headers/_tests.js

```diff
@@ -6,4 +6,12 @@ export default function (test) {
 		const headers = response.headers();
 		assert.equal(headers['permissions-policy'], 'interest-cohort=()');
 	});
+
+	test(
+		'allows headers to be sent as a Headers class instead of a POJO',
```

Comment:
this makes it sounds like you could pass either. is that correct or only a Headers instance is accepted now?

Reply:
This test was written by Rich, but I believe that both should be accepted, otherwise we'd make a breaking change

Reply:
both should be accepted, because we're just mirroring the native `fetch` API (until this PR, i didn't realise you could pass a `Headers` object; turns out `fetch` accepts either)